### ISSUE TITLE
Can't save value with quotes

### ISSUE THREAD

**Issue by davideserafini** [bluzelle/blzjs#12]
Hello team!

**Problem**
Using a value with quotes in the create or update methods returns "Invalid chain id"
This is happening with any type of quotes, escaped or not

**Code to reproduce**
This is the smallest code to reproduce, using crud.js in the samples directory.

```
const { bluzelle } = require('../src/main.js');

const main = async () => {
  var bz = await bluzelle({
      address: 'bluzelle1upsfjftremwgxz3gfy0wf3xgvwpymqx754ssu9',
      mnemonic: 'around buzz diagram captain obtain detail salon mango muffin brother morning jeans display attend knife carry green dwarf vendor hungry fan route pumpkin car',
      endpoint: "http://testnet.public.bluzelle.com:1317",
      chain_id: "bluzelle"
  });

  // Any of these doesn't work
  var res = await bz.update('test', '"', {'max_gas': '', 'max_fee': '', 'gas_price': '20'});
  //var res = await bz.update('test', '\"', {'max_gas': '', 'max_fee': '', 'gas_price': '20'});
  //var res = await bz.update('test', '{"test": "test"}', {'max_gas': '', 'max_fee': '', 'gas_price': '20'});
  console.log(typeof res != 'undefined' ? res : "success");
}
```

**Any other info**
I've already troubleshot a bit this with @njmurarka via email

**Comment by paularchard** [comment]
Issue found and fixed. Will be merged once verified by QA.

**Comment by paularchard** [comment]
Fixed by commit f6d03e04b5d8afc96577da4a7c20fe8fe08eecbc
